embulk-input-oracle/src/test/java/org/embulk/input/oracle/OracleTests.java

```diff
@@ -33,8 +32,9 @@ public static void execute(TestingEmbulk embulk, String sql) throws IOException
         String user = config.get(String.class, "user");
         String password = config.get(String.class, "password");
         String database = config.get(String.class, "database");
+
         ProcessBuilder pb = new ProcessBuilder(
-                "SQLPLUS",
+                "sql",
```

Comment:
Tried it and works ok. I think this is an issue when running sqlcl on windows (https://community.oracle.com/thread/3974824)